p2p/raintree/router.go

```diff
@@ -282,6 +298,81 @@ func (rtr *rainTreeRouter) Size() int {
 	return rtr.peersManager.GetPeerstore().Size()
 }
 
+// handleStream ensures the peerstore contains the remote peer and then reads
+// the incoming stream in a new go routine.
+func (rtr *rainTreeRouter) handleStream(stream libp2pNetwork.Stream) {
+	rtr.logger.Debug().Msg("handling incoming stream")
```

Comment:
Oh neat, thanks for sharing! :raised_hands:

Reply:
I took a look at both `#handleStream()` and `#readStream()` (assuming that's what you were referring to) and it looks like this line (:304) is the only one that's not an error level log. If there were several debug level logs, I agree that it would be worth it but am not convinced it adds any value at the moment. Will definitely keep this API and use case in mind though in the future! :wink: